pkg/cloud/services/ec2/gateways.go

```
@@ -47,7 +48,7 @@ func (s *Service) reconcileInternetGateways() error {
 
 		ig, err := s.createInternetGateway()
 		if err != nil {
-			return nil
```

Not related to PR. I assumed we'd want to return the error